pyswarms/backend/operators.py

```diff
@@ -17,6 +17,9 @@
 from ..utils.reporter import Reporter
 from .handlers import BoundaryHandler, VelocityHandler
 from functools import partial
+from multiprocessing import Process, Pipe, Pool
```

Comment:
I think you can remove Pool from here. Also, check if you still need the import below.